sympy/stats/rv.py

```diff
@@ -1565,12 +1608,24 @@ def sample(self, size=(), library='scipy', seed=None):
             else:
                 rand_state = seed
             samps = do_sample_numpy(self, size, rand_state)
+
         elif library == 'pymc3':
             from sympy.stats.sampling.sample_pymc3 import do_sample_pymc3
             import pymc3
+            #size accepts only int in case of pymc3
+            #see https://github.com/sympy/sympy/pull/21590#discussion_r649783860
+
+            size = list(size)[1:]
+            if size == []:
+                size=1
+            elif len(size) == 1:
+                size = size[0]
+            else:
+                raise TypeError("Invalid value for `size` in pymc3. Must be int")
```

Comment:
This block of code appears in at least 3 places. Is there not a better place to add this?

Reply:
There may be a way to add it only in `sample_iter`. But that causes problems in matrix_distributions and joint_rv in this part. Currently, I think that this is the best possible option for now to prevent break working code.
https://github.com/sympy/sympy/blob/a8299e5d3a4309946439dd440421c2a8d7b91ae9/sympy/stats/joint_rv.py#L239-L240